spec/unit/form_spec.rb

```diff
@@ -116,6 +116,13 @@ def self.name
     end
 
     it 'raises error when attribute is in conflict with form interface' do
+      pending <<-PENDING.strip_heredoc
+      I've updated the builder to _skip_ already existing methods, rather
+      than raising a warning.
+      1) It's needed for allowing input to be called multiple times
+      2) If you've already defined a method, good on ya?
+      DISCUSS
+      PENDING
```

Comment:
Yes this sounds like a good plan
On Tue 24 Mar 2015 at 15:04 Chris Flipse <notifications@github.com> wrote:

> In spec/unit/form_spec.rb
> <https://github.com/rom-rb/rom-rails/pull/34#discussion_r27029697>:
>
> > @@ -116,6 +116,13 @@ def self.name
> >      end
> >
> >      it 'raises error when attribute is in conflict with form interface' do
> > +      pending <<-PENDING.strip_heredoc
> > +      I've updated the builder to _skip_ already existing methods, rather
> > +      than raising a warning.
> > +      1) It's needed for allowing input to be called multiple times
> > +      2) If you've already defined a method, good on ya?
> > +      DISCUSS
> > +      PENDING
>
> I'd consider the chance of anyone coming from a rails background using
> attributes with those names in a rails app to be vanishingly small.
>
> How about a check against a list of reserved methods to catch that case,
> then?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rom-rb/rom-rails/pull/34/files#r27029697>.
>